default.json

```diff
@@ -105,7 +105,7 @@
   "ignoreNpmrcFile": true,
   "lazyGrouping": false,
   "postUpdateOptions": ["yarnDedupeHighest"],
-  "prConcurrentLimit": 2,
```

Comment:
`2` was a little aggressive for majority of our repos, where branch builds take <3 minutes and we occasionally fell behind on our Upkeep ⭐️

This is easily overwritten on an individual repo basis:

```json
{
  "extends": ["github>seek-oss/rynovate"],
  "prConcurrentLimit": 2
}
```